HPXMLtoOpenStudio/resources/electric_panel.rb

```diff
@@ -0,0 +1,245 @@
+# frozen_string_literal: true
+
+# Collection of methods related to electric panel load calculations.
+module ElectricPanel
```

Comment:
Can we change the name from `electric panel` to `electrical panel` in the code? I think the latter is the more standard term.

Reply:
For better or worse it appears to be `ElectricPanel` in the HPXML schema, so we should probably have everything consistent with that.